classes/announcement/Collector.php

```diff
@@ -124,6 +131,21 @@ public function offset(?int $offset): self
         return $this;
     }
 
+    /**
+     * Order the results
+     *
+     * Results are ordered by the date posted by default.
+     *
+     * @param string $sorter One of the self::ORDERBY_ constants
+     * @param string $direction One of the self::ORDER_DIR_ constants
+     */
+    public function orderBy(?string $sorter, string $direction = self::ORDER_DIR_DESC): self
+    {
+        $this->orderBy = $sorter;
+        $this->orderDirection = $direction;
+        return $this;
```

Comment:
(aside, but I wish they would adopt [this RFC](https://wiki.php.net/rfc/this_return_type), and go one further by making the `return $this;` implicit when no return was coded.)